aether/sdk/conf/settings.py

```diff
@@ -281,16 +294,21 @@ def get_required(name):
 
     CACHEOPS = {
         # users and roles
-        'auth.*': {},
-        'authtoken.*': {},
+        'auth.user': {
```

Comment:
The * references weren't being properly understood by cacheops, so I made them explicit.

Reply:
Ah, good point.